Panic in frostfs-cli 'container create' command

## Current Behavior
Panic in frostfs-cli container create command

## Steps to Reproduce (for bugs)
1. Following command leads to panic:
```
COMMAND: frostfs-cli --config /jenkins/workspace/sbercloud_test_nightly_failover/tmp.kBQ9YsJSH5/frostfs-testcases/wallet_config.yml container create --rpc-endpoint '172.26.160.32:8080' --wallet '/jenkins/workspace/sbercloud_test_nightly_failover/tmp.kBQ9YsJSH5/frostfs-testcases/TemporaryDir/7da1b77d-aefa-473b-82ab-40c34d47fe6c.json' --basic-acl '0FBFBFFF' --await --policy 'REP 2 IN X CBF 2 SELECT 2 FROM * AS X'
RETCODE: 2

STDOUT:
panic: empty attribute value

goroutine 1 [running]:
github.com/TrueCloudLab/frostfs-sdk-go/container.(*Container).SetAttribute(0xd4a4c2?, {0xd4e65b?, 0xc000345ae0?}, {0x0?, 0xc00036f4a0?})
	github.com/TrueCloudLab/frostfs-sdk-go@v0.0.0-20221214065929-4c779423f556/container/container.go:300 +0x2a5
github.com/TrueCloudLab/frostfs-sdk-go/container.WriteDomain(0x4c46768faa210a18?, {{0x0?, 0x4c46768faa210a18?}, {0x0?, 0x4c46768faa210a18?}})
	github.com/TrueCloudLab/frostfs-sdk-go@v0.0.0-20221214065929-4c779423f556/container/container.go:461 +0x4e
github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/modules/container.parseAttributes(0xc00036f4a0?, {0x0, 0x0, 0x0?})
	github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/modules/container/create.go:223 +0x135
github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/modules/container.glob..func1(0x1594820?, {0xd42b03?, 0xb?, 0xb?})
	github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/modules/container/create.go:84 +0x277
github.com/spf13/cobra.(*Command).execute(0x1594820, {0xc0002f8420, 0xb, 0xb})
	github.com/spf13/cobra@v1.6.1/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x1591740)
	github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.6.1/command.go:968
github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/modules.Execute()
	github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/modules/root.go:55 +0x25
main.main()
	github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/main.go:6 +0x17

STDERR:

Start / End / Elapsed	 02:05:23.861946 / 02:05:24.232256 / 0:00:00.370310
```
## Regression
No
## Version
```0.0.1-199-g6d4f48f3-dirty```

## Your Environment
Server setup and configuration:
Virtual, 4 servers, 4 SN, 4 http qw, 4 s3 gw

Operating System and version (uname -a):
linux vedi 5.10.0-16-amd64 https://github.com/nspcc-dev/neofs-node/issues/1 SMP Debian 5.10.127-1 (2022-06-30) x86_64 GNU/Linux

--- a/cmd/frostfs-cli/modules/container/create.go
+++ b/cmd/frostfs-cli/modules/container/create.go
@@ -7,6 +7,7 @@ import (
 	"strings"
 	"time"
 
+	containerApi "github.com/TrueCloudLab/frostfs-api-go/v2/container"
 	internalclient "github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/internal/client"
 	"github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/internal/common"
 	"github.com/TrueCloudLab/frostfs-node/cmd/frostfs-cli/internal/commonflags"
@@ -217,10 +218,12 @@ func parseAttributes(dst *container.Container, attributes []string) error {
 		container.SetName(dst, containerName)
 	}
 
-	var domain container.Domain
-	domain.SetName(containerNnsName)
-	domain.SetZone(containerNnsZone)
-	container.WriteDomain(dst, domain)
+	if containerNnsName != "" {
+		dst.SetAttribute(containerApi.SysAttributeName, containerNnsName)
+	}
+	if containerNnsZone != "" {
+		dst.SetAttribute(containerApi.SysAttributeZone, containerNnsZone)
+	}
 
 	return nil
 }

